pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go

```diff
@@ -197,6 +197,11 @@ func makePlan(
 			execinfrapb.ProcessorCoreUnion{StreamIngestionFrontier: streamIngestionFrontierSpec},
 			execinfrapb.PostProcessSpec{}, streamIngestionResultTypes)
 
+		for src, dst := range streamIngestionFrontierSpec.SubscribingSQLInstances {
+			log.Infof(ctx, "Physical Replication Src-Dst Pair: Node Pair List: %s:%d",
```

Comment:
[nit] I think we usually use lowercase log message.

Reply:
Done.